cmd/ocm/create/cluster/cmd.go

```diff
@@ -26,14 +26,13 @@ import (
 	"github.com/spf13/cobra"
 	"k8s.io/apimachinery/pkg/util/sets"
 
-	"github.com/openshift-online/ocm-cli/pkg/arguments"
 	c "github.com/openshift-online/ocm-cli/pkg/cluster"
 	"github.com/openshift-online/ocm-cli/pkg/ocm"
 )
 
 var args struct {
-	parameter          []string
-	header             []string
+	dryRun bool
+
```

Comment:
redundant line

Reply:
It's a "how to create" flag, I deliberately left empty lines to separate it from "what cluster to create".